src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java

```diff
@@ -197,10 +204,15 @@ public BufferedImage update(PlayerSetupInfo info)
    * Draws little white boxes around all the currently-highlighted player's properties.
    * @param distance - defines how far the boxes should be from the property edge (serves to highlight the new properties when switching between players)
    */
-  private static void drawPlayerPropertyHighlights(Graphics g, int baseX, int baseY, MapInfo mapInfo, double scale, int faction, int distance)
+  private static void drawPlayerStuffHighlights(Graphics g, int baseX, int baseY, MapInfo mapInfo, double scale, int faction, int distance)
```

Comment:
Yes. Stuff.

Reply:
`drawPlayerPropertyAndUnitHighlights` was just a smidge too long for my liking.